frontend/src/components/Charts/HoldingTimeSeriesChart.vue

```diff
@@ -234,6 +236,10 @@ const scaleClicked = async (label: string) => {
   await reloadData();
 };
 
+onBeforeMount(async () => {
+  await reloadData();
+});
```

Comment:
Since we are using Suspense, the onBeforeMount is not needed. 
```suggestion
  await reloadData();
```